src/structs/field-shorthand.md

```diff
@@ -32,6 +32,20 @@ impl Person {
         Self { name, age }
     }
 }
+```    
+* Implement a `default` setting for the struct.
```

Comment:
I don't think there is a concept of a "default setting" in Rust — we should just talk about the `Default` trait instead:

```suggestion
* Implement the `Default` trait for the struct.
```